ledger/src/blockstore.rs

```diff
@@ -515,21 +515,32 @@ impl Blockstore {
         }
     }
 
+    /// Whether to disable compaction in [`compact_storage`], which is used
+    /// by the ledger cleanup service and [`backup_and_clear_blockstore`].
+    ///
+    /// Note that this setting is not related to the RocksDB's background
+    /// compaction.
+    ///
+    /// To disable RocksDB's background compaction, open the Blockstore
+    /// with AccessType::PrimaryOnlyForMaintenance.
     pub fn set_no_compaction(&mut self, no_compaction: bool) {
         self.no_compaction = no_compaction;
     }
 
+    /// Deletes the ledger store at the specified path.
```

Comment:
nit: I admittedly use ledger/blockstore interchangeably, but I think we should say "Deletes the blockstore" to be consistent with terminology elsewhere (blockstore is a subset of ledger, and namely, the content in `rocksdb` folder)